src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -416,6 +416,26 @@
         "uriBaseId": [ "uri" ]
       }
     },
+    "fileLocation": {
+      "description": "Specifies the location of a file.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+
+        "uri": {
+          "description": "A string containing a valid relative or absolute URI.",
+          "type": "string",
+          "format": "uri"
+        },
+
+        "uriBaseId": {
+          "description": "A string which indirectly specifies the absolute URI with respect to which a relative URI in the \"uri\" property is interpreted.",
+          "type": "string"
+        }
+      },
+
+      "required": [ "uri" ]
+    },
```

Comment:
Oh, hah, I didn't realize we didn't have the `fileLocation` type in this branch yet. That's fine.